app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx

```diff
@@ -55,6 +55,9 @@ import {
 import PayloadConfigModal from 'components/PipelineTriggers/PayloadConfigModal';
 import PipelineCompositeTriggerRow from './PipelineCompositeTriggerRow';
 
+const colors = require('styles/colors.scss');
+const toolTipBackground = colors.blue07;
```

Comment:
Can you get this some way that doesn't use colors.scss?

Reply:
updated, thanks!